client/src/features/users/Auth/SignupNonProfit.tsx

```diff
@@ -43,84 +32,87 @@ const useStyles = makeStyles((theme: Theme) => ({
   arrow: { cursor: 'pointer' },
 }));
 
-interface SignupData {
-  org_name: string;
-  city: string;
-  state: string;
-  ein: string;
-  tax_exempt_id: string;
-  nonprofit_classification: string;
-  firstName: string;
-  last_name: string;
-  role_or_title: string;
-  email: string;
-  password: string;
-  accept_terms: boolean;
-}
-
-const initialFormData: SignupData = {
-  org_name: '',
+const defaultOrg: Organization = {
+  name: '',
+  doing_business_as: '',
   city: '',
   state: '',
   ein: '',
-  tax_exempt_id: '',
+  description: '',
+  website: '',
+  address: '',
+  phone: '',
   nonprofit_classification: '',
+};
+
+const defaultUser: BaseUserEntity = {
   firstName: '',
   last_name: '',
-  role_or_title: '',
   email: '',
-  password: '',
-  accept_terms: false,
+};
+
+const defaultUserOrg: UserOrg = {
+  approvalStatus: ApprovalStatus.pending,
+  role: Role.owner,
 };
 
 function SignupNonProfit() {
   const classes = useStyles();
+  const labels = ['First Step', 'Second Step'];
+  const [activeStep, setActiveStep] = React.useState<number>(0);
+  const [org, setOrg] = React.useState<Organization>(
+    (JSON.parse(sessionStorage.getItem('org') as string) as Organization) ?? defaultOrg,
+  );
+  const [user, setUser] = React.useState<BaseUserEntity>(
+    (JSON.parse(sessionStorage.getItem('user') as string) as BaseUserEntity) ?? defaultUser,
+  );
 
-  const [formData, setFormData] = React.useState(initialFormData);
-  const [pageNum, setPageNum] = React.useState(1);
-
-  const handleFieldChange = (evt: React.ChangeEvent<HTMLInputElement>) => {
-    const { name, value, type } = evt.target;
+  const [userOrg, setUserOrg] = React.useState<UserOrg>(
+    (JSON.parse(sessionStorage.getItem('org') as string) as UserOrg) ?? defaultUserOrg,
+  );
 
-    setFormData((fData) => {
-      return {
-        ...fData,
-        [name]: type === 'checkbox' ? evt.target.checked : value,
-      };
-    });
+  const onChildSetParentOrg = (org: Organization): void => {
+    sessionStorage.setItem('org', JSON.stringify(org));
+    setOrg(org);
   };
 
-  //Had to make second handler for HTMLSelectElement vs HTMLInputElement
-  //checked property does not exist on HTMLSelectElement, and so general handler does not work.
-  const handleSelectChange = (evt: React.ChangeEvent<HTMLSelectElement>) => {
-    const { name, value } = evt.target;
-
-    setFormData((fData) => {
-      return {
-        ...fData,
-        [name]: value,
-      };
-    });
+  const onChildSetParentUserOrg = (userOrg: UserOrg): void => {
```

Comment:
hmm i'm not understanding the "parent" and "child" terms here?

Reply:
Parent <-> child component communication

Reply:
ok i think with the new pattern there will only be 1 state so we shouldn't really need the parent/child labeling?